api/eqip/client.go

```diff
@@ -52,25 +51,31 @@ func (c *Client) Send(action RequestBody, response Body) error {
 	}
 	err = xml.Unmarshal(buf.Bytes(), &temp)
 	if err != nil {
-		fmt.Println("Error unmarshalling xml: ", err)
-		return err
+		return fmt.Errorf("SOAP request is malformed: %s", err.Error())
 	}
+
 	resp, err := http.Post(c.endpointURL, "text/xml; charset=utf-8", bytes.NewReader(temp.XML))
-	// Regardless of what happens, we always save the response body. In the event of network http error,
-	// we set the response body to the error message
```

Comment:
Just confirming: this should no longer be the case?

Reply:
Yeah, that wasn't a design choice I agreed with and makes the semantics of what gets persisted in the database muddy. Anything other than a SOAP response will get logged like other errors.